### PR TITLE
Reducing pytest output by default and storing the debug output to a

### PR DESCRIPTION
**Pull request by ehsavoie**
pytest.log file.

### CODE REVIEW

**Comment by kabir** [comment]
@ehsavoie will this still output to the console on CI? File output is a bit painful there.

This is the command on CI: https://github.com/wise-agents/wise-agents/blob/main/.github/workflows/build-and-test-workflow-run.yml#L115

````
pytest --log-cli-level=DEBUG -m "not needsllm and not needsllama"
````

**Comment by ehsavoie** [comment]
@kabir you can still override it with the cli parameter. I set it to CRITICAL but maybe INFO would be best

File: pyproject.toml [hunk 1]
```diff
@@ -31,17 +31,21 @@ test = [
...
 ]
 
-[pytest]
-pythonpath = "src"
-
 [tool.pytest.ini_options]
 log_cli = true
-log_cli_level = "INFO"
-log_cli_format = "%(asctime)s [%(levelname)] %(message)s"
+log_cli_level = "CRITICAL"
```
Comment (maeste) [inline pyproject.toml:36]: I think INFO is more appropriate

**Review by maeste — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 1 above)
Comment (maeste) [reply pyproject.toml:36]: Hmm, maybe not... let's give it a try. Approving and merging it as is

**Review by maeste — APPROVED** [review]

**Pull request merged**